styx-common/src/main/java/com/spotify/styx/util/WorkflowValidator.java

```diff
@@ -133,6 +139,13 @@ public static Builder newBuilder(DockerImageValidator dockerImageValidator) {
       }
     });
 
+    // Check if the service account contains space
```

Comment:
Comment is not valid anymore. You can remove it

Reply:
Removed.